src/pages/Home/sections/NotableUsersSection.js

```diff
@@ -39,29 +71,34 @@ const NotableUsersSection = () => {
   const SectionTitle = () => {
     return (
       <Grid item lg={12} md={12} sm={12} className={classes.sectionTitle}>
-        <Typography  variant='h1' align='center' > See how the Civic Tech Index is used by:</Typography>
+        <Typography variant='h1' align='center'>
+          {' '}
+                    See how the Civic Tech Index is used by:
+        </Typography>
       </Grid>
     );
   };
 
   const LogoList = () => {
     const LogoItem = (props) => {
       return (
-        <Grid item  lg ={3} md={3} xs={6} style={{ height: '120px' }}>
+        <Grid item lg={3} md={3} xs={6} style={{ height: '120px' }}>
```

Comment:
md={3} once you apply to medium it applies to the large screen too,
<Grid item md={3} xs={6} style={{ height: '120px' }}>